apps/sage_intacct/queue.py

```
@@ -89,15 +96,40 @@ def schedule_journal_entries_creation(
+        now = datetime.now(tz=timezone.utc)
+
+        if task_log:
+            # if the task log is created before 2 months
+            if task_log.created_at <= now - relativedelta(months=2):
+                task_log.is_retired = True
+                task_log.save()
+                return True
+
+            # if the task log is created in the last 2 months
+            if now - relativedelta(months=2) < task_log.created_at <= now - relativedelta(months=1):
+                if now - task_log.updated_at.replace(tzinfo=timezone.utc) <= timedelta(weeks=1):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)